packages/module/src/Chatbot/Chatbot.tsx

```diff
@@ -49,10 +53,27 @@ export const Chatbot: React.FunctionComponent<ChatbotProps> = ({
       variants={motionChatbot}
       initial="hidden"
       animate={isVisible ? 'visible' : 'hidden'}
+      {...props}
     >
-      {isVisible ? children : undefined}
+      {/* Ref is intended for use with skip to chatbot links, etc. */}
+      {/* Motion.div does not accept refs */}
+      {isVisible ? (
+        <div
+          role="region"
+          aria-label={props['aria-label'] ?? 'Chatbot'}
+          className={`pf-chatbot-container pf-chatbot-container--${displayMode} ${!isVisible ? 'pf-chatbot-container--hidden' : ''}`}
+          tabIndex={0}
```

Comment:
For the container, we should do a `tabIndex={-1}` instead since we'd really more want to be able to set focus from the skip link rather than be able to focus it via keyboard navigation. Doing that will still allow stuff like VoiceOver to navigate to the container wrapper and announce its accessible name, just prevent navigating to the container wrapper as part of the normal tab order of the page.